drivers/CoreFlashMemory/source/CoreFlashManagerIS25LP016D.cpp

```diff
@@ -44,28 +44,25 @@ void CoreFlashManagerIS25LP016D::waitForChipAvailable()
 	_timer.stop();
 }
 
-void CoreFlashManagerIS25LP016D::reset()
+void CoreFlashManagerIS25LP016D::sendSimpleCommand(uint8_t command)
 {
 	waitForChipAvailable();
 	if (chipIsNotAvailable()) {
 		return;
 	}
-	_qspi.sendCommand(command::reset_enable, -1, tx_buffer, 0, rx_buffer, 0);
+	_qspi.sendCommand(command, -1, tx_buffer, 0, rx_buffer, 0);
+}
```

Comment:
ça me va bien ! 

est-ce qu'un return true/false serait pas pratique si on veut recommencer ?

```suggestion
auto CoreFlashManagerIS25LP016D::sendSimpleCommand(uint8_t command) -> bool
{
	waitForChipAvailable();
	
	if (chipIsNotAvailable()) {
		return false;
	}
	
	_qspi.sendCommand(command, -1, tx_buffer, 0, rx_buffer, 0);
	return true;
}
```

Reply:
C'était nécessaire pour bien refactor en effet :)